packages/select/src/MenuButton.tsx

```diff
@@ -81,30 +183,28 @@ const menuButtonTextStyle = css`
   max-width: 100%;
 `;
 
-const errorColor: Record<Mode, string> = {
-  [Mode.Light]: palette.red.base,
-  [Mode.Dark]: '#F97216',
+const errorColor: Record<Theme, string> = {
+  [Theme.Light]: palette.red.base,
+  [Theme.Dark]: '#FF6960', // TODO: palette.red.light1 is different in figma(#FF6960) which does not match react(#EF5752) but will be updated with this ticket https://jira.mongodb.org/browse/PD-2206
 };
 
-const menuButtonErrorStyle: Record<Mode, string> = {
-  [Mode.Light]: css`
-    border-color: ${errorColor[Mode.Light]};
+const menuButtonErrorStyle: Record<Theme, string> = {
+  [Theme.Light]: css`
+    border-color: ${errorColor[Theme.Light]};
     background-color: ${palette.white};
 
     &:hover,
     &:active {
       box-shadow: 0 0 0 3px #f9d5c5; // Between light2 & light3
```

Comment:
This can be `hoverRing.light.red`

Reply:
Hm looks like the color should be `palette.red.light1`. `hoverRing.light.red` is using `palette.red.light2`

Reply:
nvm, you're correct!